packages/flutter_test/lib/src/browser.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const bool _kIsCompiledToJavaScript = identical(0, 0.0);
```

Comment:
Would it make sense to make this a public global?  Seems pretty generally useful.

Reply:
I'm doing something similar here: https://github.com/flutter/flutter/pull/33406